website/content/docs/commands/operator/raft.mdx

```diff
@@ -184,3 +184,112 @@ Usage: vault operator raft snapshot restore <snapshot_file>
 
 	  $ vault operator raft snapshot restore raft.snap
 ```
+
+## autopilot
+
+This command groups subcommands for operators interacting with the autopilot
+functionality of the integrated Raft storage backend. There are 3 subcommands
+supported: `get-config`, `set-config` and `state`.
```

Comment:
It's useful to know (although maybe this is a thing that belongs more in the API docs) what kind of node can run these commands? Only the active? Perf standbys? DR clustes?

Reply:
In terms of active vs standbys: we strive to ensure that the user doesn't need to know, i.e. if a standby can't handle a certain request, it should forward transparently to the active node.

As to the DR cluster question: this is something we don't document elsewhere AFAIK, primarily because there's so few endpoints supported in DR that it probably makes more sense to highlight the ones that *do* work on DR rather than add a note to 99% of the APIs saying "this doesn't work on DR."